README improvements (#368)
gbdev/pandocs: README.MD

- [The wiki](https://github.com/gbdev/pandocs/wiki), containing various rules and contribution guidelines you will have to follow when proposing changes. Some RFCs and discussions detailing the document scope are linked there as well.
- [The project board](https://github.com/gbdev/pandocs/projects/1), for a general picture of the roadmap and the ongoing efforts.

> - [The issues](https://github.com/gbdev/pandocs/issues), where we discuss what needs to be worked on, and how. If you are looking into investigating Game Boy unknown behaviours, check the Issues labeled with ["research"](https://github.com/gbdev/pandocs/issues?q=is%3Aissue+is%3Aopen+label%3Aresearch).

Reviewer: Should we document more of the labels? Also, maybe splitting this info in a `CONTRIBUTING.md` file?

Author: I think it's okay to just mention *some* of them here and everything in the CONTRIBUTING file. On this topic: https://github.com/gbdev/pandocs/issues/370

Be aware of the following caveats:

- `docs/html/` contains only partially processed files and it's also the folder that gets served by `mdbook serve`, so you will see some unprocessed custom markup if you visit the endpoint exposed by mdbook's development web server (:3000).

> As a workaround, you can manually serve the file in the `docs/pandocs/` with the web server of your choice (e.g. you can run `python3 -m http.server` from the `docs/pandocs` folder).

Reviewer: Careful that without two spaces of indentation, this is a paragraph that breaks the list, not a paragraph within the list item.

Author: Oops, you're right. I should've fixed.

These are rendered as "info boxes".

- `type` must be `tip`, or `warning`.

> - `HEADING` can contain spaces and will be the title of the box. It can be "Warning", "Tip" or something illustrative of the phenomenon described in the box.

Reviewer: ```suggestion
- `HEADING` can contain spaces and will be the title of the box. It should be `Warning`, `Tip`, or something illustrative of what the box contains.
```
